cms/tests/test_wizards.py

```diff
@@ -154,74 +163,109 @@ def test_endpoint_auth_required(self):
 
 class TestWizardPool(WizardTestMixin, CMSTestCase):
 
-    def test_discover(self):
-        wizard_pool._reset()
-        self.assertFalse(wizard_pool._discovered)
-        self.assertEqual(len(wizard_pool._entries), 0)
-        wizard_pool._discover()
-        self.assertTrue(wizard_pool._discovered)
+    def tearDown(self):
+        # Clean up in case anything has been removed or added to the
+        # registered wizards, so other tests don't have problems
+        extension = apps.get_app_config('cms').cms_extension
+        expected_wizards = {
```

Comment:
this means we would have to keep this dict updated for every new wizard in core or tests.
I think we can just trigger a reload of the apps no?

Reply:
There could be side effects to that (such as getting an exception for registering a wizard twice, but there could be more stuff like that once more things are integrated with app registration). But I think there are other ways I can be more generic here.